r124/instream.csh

```diff
@@ -21,6 +21,7 @@
 # jnlunxpcterr     [nars]  Test that MUPIP JOURNAL -EXTRACT does not issue JNLUNXPCTERR error in the face of concurrent udpates
 # ydb293	   [vinay] Tests the update process operates correctly with triggers and SET $ZGBLDIR
 # ydb297	   [vinay] Demonstrates LOCK commands work correctly when there are more than 31 subscripts that hash to the same value
+# ydb312_gtm8182f  [jake]  Tests a fix for an incorrectly issued REPLINSTMISMTCH error upon accessing a locked variable within an additional instance accessed by a process
```

Comment:
upon accessing .... -> in a LOCK command with extended references in a process that accesses multiple instances